psrp/script.yaml

```
@@ -236,8 +236,8 @@
     Estas pessoas são perigosas, você vai precisar de ajuda.<wait more>
     Ouvi histórias de um guerreiro chamado Tylon, que poderia ser um aliado.<line><line>
     Queria contar com sua ajuda, mas ele está desaparecido.<wait more>
-    Você tem que encontrá-lo, Alisa.<line><line>
-    Encontre Tylon, e derrote LaShiec. Tragam a paz de volta a Algol.<wait more>
+    Alisa, você precisa encontrar Tylon.<line><line>
+    Encontre-o, e derrote LaShiec. Juntos, vocês podem trazer a paz de volta a Algol.<wait more>
```

No need for this comma.
```suggestion
    Encontre-o, e derrote LaShiec. Juntos vocês podem trazer a paz de volta a Algol.<wait more>
```

---

> No need for this comma.

i changed this text.